src/index.js

```diff
@@ -1,19 +1,22 @@
 /* Imports */
 import './styles.css';
+import {
+  deployList, addToList, editList, removeList,
+} from './modules/functionality.js';
 
 const activitiesToDo = [
   {
-    description: 'First To Do List',
+    description: 'Reading JS Documentation',
     completed: false,
     id: 1,
   },
   {
-    description: 'Second To Do List',
+    description: 'Reading Bootstrap Documentaion',
     completed: false,
     id: 2,
   },
```

Comment:
- [ ] Please remove the hardcoded tasks 
![image](https://user-images.githubusercontent.com/62705947/229165385-c5106638-2610-4f73-b7f5-0fa0fed6af83.png)

Reply:
- [ ] Your add, delete and edit functions seem correct but you have an error on runtime which is allowing me to do neither of those functionalities. Please check into this, it may have problem with webpack